src/System/Router/Boot.tsx

```diff
@@ -123,6 +125,19 @@ export const Boot = track(undefined, {
   )
 })
 
+const EnvironmentProvider: FC<{ environment: Environment }> = ({
+  children,
+  environment,
+}) => {
+  if (process.env.NODE_ENV === "test") return <>{children}</>
```

Comment:
Why not provide the environment in tests? Is it because we wrap the setupTestWrapper util in an environment provider?

Reply:
It _should_ already have been provided by `MockBoot` — I couldn't really make sense of it and it didn't seem to matter much since nothing really needed it. It's possible that as we move over to relay-hooks we'll have to address though.

Reply:
Yah we should avoid this if possible since we're rapidly moving to relay hooks! Mind dropping it into mock boot? It seems pretty harmless. I can imagine this will catch some unsuspecting dev later

Reply:
I'll make a ticket to take a closer look; but that's the thing: `MockBoot` _is_ `Boot` — it should have just worked.

Reply:
ahhh i see. Interesting, yah. Ty 👍